lesson-3/Berman-lesson3/About/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>About page !</title>
+  </head>
+  <body>
+
+
+    <h1>About</h1>
+
+    <img src='../images/rachel.jpg'width='300px'/>
+
+    <ul>
+      <li><a href="../../Berman-lesson3/">Home</a></li>

```

Comment:
So close! You went one step too far back here. You only need one `../` to get back to the main folder. Your main folder in this case is `Berman-lesson3`.